doc-Managing_Authentication/topics/External_AD.adoc

```diff
@@ -0,0 +1,158 @@
+services = nss, pam, ifp
+default_domain_suffix = example.com
+
+[nss]
+homedir_substring = /home
+
+[pam]
+default_domain_suffix = example.com
+
+[ifp]
+default_domain_suffix = example.com
+allowed_uids = apache, root
+user_attributes = +mail, +givenname, +sn, +displayname, +domainname
+----
++
+. Restart and enable the `sssd` service:
++
+----
+# systemctl restart sssd
+# systemctl enable sssd
+----
++
+//Do we need the upstream step for chgrp apache? Added just in case
+. Make sure the Kerberos keytab created by `realm join` above is readable by Apache:
++
+----
+# chgrp apache /etc/krb5.keytab
+# chmod 640 /etc/krb5.keytab
+----
++
+. Copy the following `httpd` configuration files into the correct respective directories with the following commands:
++
+----
+# TEMPLATE_DIR="/opt/rh/cfme-appliance/TEMPLATE"
+# cp ${TEMPLATE_DIR}/etc/pam.d/httpd-auth /etc/pam.d/httpd-auth
+# cp ${TEMPLATE_DIR}/etc/httpd/conf.d/manageiq-remote-user.conf /etc/httpd/conf.d/
+# cp ${TEMPLATE_DIR}/etc/httpd/conf.d/manageiq-external-auth.conf.erb /etc/httpd/conf.d/manageiq-external-auth.conf
+----
++
+. Edit the `/etc/httpd/conf.d/manageiq-external-auth.conf` file to point to your Kerberos domain hosted by your Active Directory domain by adding the lines for `KrbAuthRealms`, `Krb5KeyTab` and `KrbServiceName` for your environment:
+// do we edit only the lines that say "add line?"
++
+----
+<Location /dashboard/kerberos_authenticate>
+  AuthType           Kerberos
+  AuthName           "Kerberos Login"
+  KrbMethodNegotiate On
+  KrbMethodK5Passwd  Off
+  KrbAuthRealms      example.com     
+  Krb5KeyTab         /etc/krb5.keytab 
+  KrbServiceName     Any                   
+  Require            pam-account httpd-auth
+
+  ErrorDocument 401  /proxy_pages/invalid_sso_credentials.js
+</Location>
+----
++
+. Set the appropriate SELinux booleans:
++
+----
+# setsebool -P allow_httpd_mod_auth_pam on
+# setsebool -P httpd_dbus_sssd          on
+----
++
+. Restart and enable the `httpd` service:
++
+----
+# systemctl restart httpd
+# systemctl enable httpd
+----
+
+Complete authentication setup by configuring the following from the {product-title_short} user interface:
+
+. Log in to the web interface as an administrative user.
+. Navigate to the settings menu, then menu:Configuration[Authentication].
+. Select a *Session Timeout* if required.
+. Select *External (httpd)* as the authentication *Mode*.
+. Select *Enable Single Sign-On* to allow single sign-on using Kerberos tickets from client machines that authenticate to the same Active Directory domain as the appliance.
+. In the *Role Settings* area, select *Get User Groups from External Authentication (httpd)*.
+. Click *Save*.
+//add screenshot from article.
+
+[IMPORTANT]
+======
+* The configuration steps for the {product-title_short} user interface must be completed on each user interface and web services enabled appliance (appliances with `user_interface` or `web_services` server roles enabled).
```

Comment:
I think "for the {product-title_short} user interface" can be eliminated here? > The configuration steps must be completed on each user interface and web services enabled appliance (appliances with `user_interface` or `web_services` server roles enabled).

Reply:
I've rearranged this a little and moved this idea to above the procedure, and have removed some of those repetitive words.